content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,237 @@
-   To work with the *FEFO* strategy, activate the *Expiration Dates* feature.
+   To work with the FEFO strategy, also activate the Expiration Dates feature.
 
-Next, you need to define your removal strategy, on *Product Categories* via
-:menuselection:`Inventory --> Configuration --> Product Categories`.
+Next, go to :menuselection:`Inventory --> Configuration --> Product Categories` to define the
+removal strategy on a product category.
 
-.. image:: removal/first-in-first-out.png
+.. image:: removal/product-category-removal.png
    :align: center
-   :alt: force removal strategy set up as first in first out.
+   :alt: :alt: Removal strategy on a product category.
 
 FIFO (First In, First Out)
 --------------------------
 
-As said, a *FIFO* strategy implies that products stocked first move out first. Companies should use
-that method if they are selling products with short demand cycles, such as clothes, and to ensure
-they are not stuck with outdated styles in stock.
+As explained, a :guilabel:`FIFO` removal strategy implies that products stocked first move out
+first. Companies should use this method if they are selling products with short demand cycles, such
+as clothes, to ensure they are not stuck with outdated styles in stock.
 
-For this example, we created three lots of white shirts. Those are from the All/Clothes category,
-where we put *FIFO* as the removal strategy. In our stock location (WH/Stock), we now find the
-three lots available.
+In this example, there are three lots of white shirts. The shirts are from the All/Clothes category,
+where :guilabel:`FIFO` is set as the removal strategy. In the
+:guilabel:`Inventory Valuation Report`,  the three different receipts are listed with the amounts.
 
 .. image:: removal/inventory-valuation.png
    :align: center
-   :alt: view of the white shirt lots inventory valuation.
+   :alt: View of the lots of white shirts in the inventory valuation report.
 
-The lot 000001 contains 5 shirts, 000002 contains 3 shirts, and 000003 contains 2. As it can be
-seen above, 000001 has entered the stock first. Let’s create a sale order of six white shirts
-to check that products from that lot are the first ones to move out.
+Lot 000001 contains five shirts, lot 000002 contains three shirts, and lot 000003 contains two
+shirts.
 
-On the delivery order linked to the picking, you can see that the oldest lot numbers have been
-reserved thanks to the *FIFO* strategy.
+As seen above, lot 000001 entered the stock first. :guilabel:`Create` and :guilabel:`Confirm` a
+sales order for six white shirts. All five shirts from lot 000001 and one shirt from lot 000002 will
+be selected to be sent to the customer.
 
-.. image:: removal/reserved-lots-FIFO.png
-   :align: center
-   :alt: two lots being reserved for sell with the FIFO strategy.
+Once the sales order is confirmed, the delivery order will be created and linked to the picking, and
+the oldest lot numbers will be reserved thanks to the FIFO strategy.
```

Comment:
I think to beef up this section we could walk through the whole SO process. After we create and confirm it, go to the picking, then to the warehouse delivery form, and deliver/invoice for the products.